models/articles.models.js

```diff
@@ -17,14 +17,51 @@ exports.selectArticleById = (article_id) => {
   });
 };
 
-exports.selectArticles = () => {
-  return db
-    .query(
-      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at,articles.votes, articles.article_img_url, count(comments) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+exports.selectArticles = (order, sort_by, topic) => {
+  const queryValues = [];
+  if (!order) {
+    order = "desc";
+  }
+  if (!sort_by) {
+    sort_by = "created_at";
+  }
```

Comment:
This can also be handled more concisely with a statement such as
order = "desc", ...

In line 20, this allows 'order' to default to 'desc', but if an argument is provided, it will use that instead.